docs/kubernetes/operator/instrumenting-applications.md

```diff
@@ -0,0 +1,159 @@
+# Instrumenting applications with EDOT on Kubernetes
+
+Elastic Distribution of OpenTelemetry (EDOT) extends [OpenTelemetry language SDKs](https://opentelemetry.io/docs/languages/) for multiple languages:
+
+* [EDOT Java](https://github.com/elastic/elastic-otel-java)
+* [EDOT .NET](https://github.com/elastic/elastic-otel-dotnet)
+* [EDOT Node.js](https://github.com/elastic/elastic-otel-node)
+* [EDOT Python](https://github.com/elastic/elastic-otel-python)
+* [EDOT PHP](https://github.com/elastic/elastic-otel-php/)
+
+This section provides guidance and examples for applications instrumentation in a Kubernetes environment for all supported languages.
+
+In Kubernetes environments with the OpenTelemetry Operator, [**automatic (or zero-code) instrumentation**](https://opentelemetry.io/docs/kubernetes/operator/automatic/) simplifies the process by automatically injecting and configuring instrumentation libraries into the targeted Pods.
```

Comment:
we use `zero-code` instrumentation wording in some places, but in other docs and later in the same doc we also mention it as `auto-instrumentation` (11 times).

Also in the Opentelemetry docs this is called `Auto-instrumentation` the majority of times (https://opentelemetry.io/docs/kubernetes/operator/automatic/).

Don't you think the sentence automatic (or zero-code) instrumentation gives better background for the rest of the document? If we just leave `zero-code` it will be the only reference in the doc, as after that it's always referred as `auto-instrumentation` (the same as in the opentelemetry docs).

please share your thoughts :)

Reply:
I think it makes sense to refer to both but either as auto-instrumentation or zero-code, my comments here was more about the word automatic being used twice in the same sentence.

Reply:
thanks for the clarification! And you are right. Updating it...